public-ui/src/app/data-browser/components/survey-answer/survey-answer-react.component.tsx

```diff
@@ -0,0 +1,255 @@
+                            : 'right'} /> : <ClrIcon className='survey-row-icon' shape='bar-chart' />}
+                </div>
+            </div>
+        </div >
+            {this.state.drawerOpen ? <div className='survey-row-expansion'>
+                {(this.hasSubQuestions === '1' && this.state.subQuestion) ? <React.Fragment>
+                    <h6 className='sub-question-text'><ClrIcon shape='child-arrow' />{this.state.subTitle}</h6>
+                    <div className='survey-sub-table'>
+                        {/* tslint:disable-next-line: no-use-before-declare  */}
+                        <SurveyAnswerReactComponent
+                            particpantCount={this.props.countValue}
+                            question={this.state.subQuestion} isCopeSurvey={this.props.isCopeSurvey} />
+                    </div>
+                </React.Fragment> :
+                    <h5>graph-component</h5>
+                }
+            </div> : undefined}
+        </React.Fragment>;
+    }
+});
+
+interface Props {
+    isCopeSurvey: boolean;
+    question: any;
+    particpantCount: any;
+}
+
+export const SurveyAnswerReactComponent = (class extends React.Component<Props, {}> {
+    constructor(props: Props) {
+        super(props);
+    }
+
+    isSubTable = this.props.particpantCount ? true : false;
+
+    render(): any {
+        return <React.Fragment><style>{styleCss}</style>
+            <div className='survey-tbl'>
+                <div className='survey-tbl-r survey-tbl-head'>
+                    <div className='info-text first survey-tbl-d'>
+                        Answer
+                    </div>
+                    <div className='survey-tbl-r-group survey-tbl-r-group-style '>
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> : <span><React.Fragment>Concept Code</React.Fragment>
+                                <TooltipReactComponent tooltipKey='conceptCodeHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' /></span>
+                            }
+                        </div >
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> : <span><React.Fragment>Participant Count</React.Fragment>
+                                <TooltipReactComponent tooltipKey='surveyParticipantCountHelpText' label='test' searchTerm='test' action='Survey Page Tooltip' /></span>
+                            }
+                        </div >
+                        <div className='info-text survey-tbl-d display-body'>
+                            {this.props.isCopeSurvey ? <h1>testIs cope</h1> :
+                                <span>{this.isSubTable ? <React.Fragment>% Answered out of {this.props.particpantCount}</React.Fragment> :
+                                    <React.Fragment>% Answered </React.Fragment>}</span>
```

Comment:
We should remove the `isSubTable` var and just use `particpantCount` here (may need to add `!!` if it renders the count).
Also can use strings here instead of `<React.Fragment>`:
```suggestion
                                <span>
                                    {!!particpantCount ? `% Answered out of ${particpantCount}` : '% Answered'}
                                </span>
```